fdbclient/Schemas.cpp

```diff
@@ -24,6 +24,28 @@
 const KeyRef JSONSchemas::statusSchema = LiteralStringRef(R"statusSchema(
 {
    "cluster":{
+	  "storage_wiggler": {
+         "primary": {
+          	"last_round_start": 1,
```

Comment:
These names should be more indicative of their units.  Timestamp fields should end in `_timestamp`, versions in `_versions`.   For duration, we use `_seconds` in several other places but `_duration` is not used except here, it would be better to be consistent.

Reply:
I missed that I should do convert from timestamp to string representation here. So the relative time fields should be strings rather than the timestamp.